gamemode/config/config.lua

```diff
@@ -500,3 +500,9 @@ Hungermod module
 GM.Config.hungerspeed = 2
 -- starverate <Amount> - How much health that is taken away every second the player is starving  (3 is the default).
 GM.Config.starverate = 3
+
+--[[---------------------------------------------------------------------------
+DO NOT TOUCH THIS (unless you want broken shit!)
+---------------------------------------------------------------------------]]
+-- Prepare minHitDistance for DistToSqr
+GM.Config.minHitDistance = GM.Config.minHitDistance ^ 2
```

Comment:
No. No, no, no, no, no, no, no. 

Don't make config options for things you don't want people to configure. That should be a local variable if used in one place, and some function if used in more than one file. 

Also, never write `X ^ 2`.It's really bad practice. Write  `X * X` instead. It's about the same performance difference as square root, no square root. Powers take more cycles than multiplications. 

No real noticeable difference in this case, but bad practice nonetheless. I'm not contradicting my own arguments here.